src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -30,12 +30,14 @@ export default (
       return { ...state, languages: action.params.languages }
     case 'PUSH_CATEGORY':
       if (state === null) {
-        throw Error('Cannot push category on not initialized cityContent')
+        // throw Error('Cannot push category on not initialized cityContent')
+        return null
```

Comment:
I'd prefere to avoid setting the `state` to `null` instead of expecting the state to be null.

Also you can remove the whole `if` here. If the state is already null then you do not need to do anything as the default case will return the `state` (which is null)

Reply:
Not sure about a better way here.